packages/@okta/vuepress-site/docs/guides/sign-into-spa-redirect/main/index.md

```diff
@@ -85,9 +85,9 @@ At this point, you can move to the next step: [Creating your app](#create-an-app
 1. Click **Create App Integration**.
 1. Select a **Sign-in method** of **OIDC - OpenID Connect**.
 1. Select an **Application type** of **Single-Page Application**, then click **Next**.
-   > **Note:** If you choose an inappropriate application type, it can break the sign-in or sign-out flows by requiring the verification of a client secret, which is something that public clients don't have.
+   > **Note:** If you choose an inappropriate app type, it can break the sign-in or sign-out flows by requiring the verification of a client secret, which is something that public clients don't have.
 1. Enter an **App integration name**.
-1. Select **Authorization Code** and **Refresh Token** as the **Grant type**. This enables the Authorization Code flow with PKCE for your application and the ability to refresh the access token when it expires without prompting the user to reauthenticate.
+1. Select **Authorization Code** and **Refresh Token** as the **Grant type**. This enables the Authorization Code flow with PKCE for your app and the ability to refresh the access token when it expires without prompting the user to reauthenticate.
```

Comment:
```suggestion
1. Select **Authorization Code** and **Refresh Token** as the **Grant type**. This enables the authorization code flow with PKCE for your app and the ability to refresh the access token when it expires without prompting the user to reauthenticate.
```

Reply:
we capitalize the name of grant type flows